src/elli/solver2x2.py

```diff
@@ -55,6 +55,16 @@ def calculate(self) -> Result:
                 )
             )
 
+        for layer in n_list:
+            if np.any(np.logical_and(layer.real > 0, layer.imag < 0)):
```

Comment:
I think we never really discussed what it should be. We both assumed that it would be an error.

However, after thinking about it I think it might be better to make it a warning. My view here is that an error should only be thrown if the problem is unrecoverable and the program cannot continue with this value. This is not the case for us. Also I find it always harsh when I program quits with an error and typically it's nicer to get weird data + a warning which tells you how to resolve the problem.

Reply:
Btw, if you make it a warning. Do we want start using [logging](https://docs.python.org/3/library/logging.html) for this stuff?

I don't know if we have some other information we display somewhere, but I think we never really outputted a warning or so...

Reply:
Ok then we agree, a warning is best. 

I never used logging, but the dedicated standard library is most of the time the best way to do something like this. I will read into it. 
Maybe this is also a good way to print debugging information instead of wrinting printing statements into the code.

Reply:
I have decided to use the warning package for now, as long as we do not use any other logging features.